src/plus/integrations/integrationService.ts

```diff
@@ -246,11 +246,9 @@ export class IntegrationService implements Disposable {
 				await openUrl(this.container.getGkDevExchangeUri(exchangeToken, `connect?${query}`).toString(true));
 			} catch (ex) {
 				Logger.error(ex, scope);
-				await env.openExternal(this.container.getGkDevUri('connect', query));
+				if (!(await env.openExternal(this.container.getGkDevUri('connect', query)))) return false;
 			}
-		} else {
-			await env.openExternal(this.container.getGkDevUri('connect', query));
-		}
+		} else if (!(await env.openExternal(this.container.getGkDevUri('connect', query)))) return false;
```

Comment:
Linting shouldn't allow this return here -- we shouldn't have single line returns outside of a single one line `if`